packages/client/src/ConfigTest.ts

```diff
@@ -20,23 +21,18 @@ const sideChainConfig = {
  */
 export const CONFIG_TEST: StreamrClientConfig = {
     network: {
-        trackers: [
-            {
-                id: '0xb9e7cEBF7b03AE26458E32a059488386b05798e8',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30301`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30301`
-            }, {
-                id: '0x0540A3e144cdD81F402e7772C76a5808B71d2d30',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30302`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30302`
-            }, {
-                id: '0xf2C195bE194a2C91e93Eacb1d6d55a00552a85E2',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30303`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30303`
-            }
-        ],
-        webrtcDisallowPrivateAddresses: false,
-        iceServers: []
+        layer0: {
+            entryPoints: [{
+                id: 'entryPointBroker',
+                type: NodeType.NODEJS,
```

Comment:
Was it so that the enum values are mainly about connectivity level: in general in browsers we have limited connectivity, and in Node we have better connectivity? 
- Does this apply e.g. situations where a user runs e.g. Raspberry Pi which is connected to a mobile network. If he used NodeJS there, do we recommend to use NODEJS and not BROWSER?
- What about other runtimes, e.g. Deno?

Or would it make sense to rename this to something else: maybe not `connectivity`: `GOOD` / `BAD`, but something similar?

Reply:
Most likely `type` will get removed in future: https://linear.app/streamr/issue/NET-990/is-peerdescriptor-browser-type-necessary. So will skip this comment.